apps/web/src/pages/auth/InvitationPage.tsx

```diff
@@ -57,18 +62,11 @@ export default function InvitationPage() {
           customDescription={
             <Center inline mb={40} mt={20}>
               <Text size="lg" color={colors.B60}>
-                <When truthy={invalidCurrentUser && !loadingAcceptInvite}>
-                  <p>The invite is not valid for the current user. Please log in with the right user.</p>
-                </When>
-
-                <When truthy={!invalidCurrentUser && !loadingAcceptInvite}>
-                  <p>Your session is currently active, use another browser or switch to incognito mode.</p>
-                  <p>Log out instead?</p>
-                </When>
```

Comment:
We don't need to show two different descriptions, which basically means the same that you have to logout.